docs/book/v3/features/error-handling.md

```diff
@@ -213,6 +213,22 @@ class LoggingErrorListenerDelegatorFactory
 }
 ```
 
+Then, enable the delegator factory by registering it in the `delegators` element of the relevant [ConfigProvider class](https://docs.laminas.dev/laminas-config-aggregator/config-providers/), as in the example below.
```

Comment:
What I don't like on the entire page: listeners should be created, configuration must be extended and now a config provider is to be extended. But where is the description in which file and under which path this should be done?
While this may be flexible and different for each application, an example must be given.

Otherwise we say: _bird eat and die anyway._

Reply:
I don't quite follow the sentence at the end, however, I appreciate your point. I can either work in the required changes in this PR, or in a subsequent one. Which would you prefer?

Reply:
Add a filename and path to your code example, everything else can be done in another pull request.

Example: https://docs.laminas.dev/laminas-form/v3/application-integration/usage-in-a-laminas-mvc-application/#register-form-and-controller

Reply:
> Add a filename and path to your code example, everything else can be done in another pull request.
> 
> Example: https://docs.laminas.dev/laminas-form/v3/application-integration/usage-in-a-laminas-mvc-application/#register-form-and-controller

That's an excellent example. Thanks, @froschdesign. Will do that a little later, this morning.

Reply:
Hey @froschdesign, I added the example that I thought might most often come to mind, that being the `getDependencies` method in _src/App/src/ConfigProvider.php_. The `App` module is the one that comes with the Mezzio Skeleton project and used most often in the docs, iirc.

I italicised the path as I picked up that convention while working at Twilio. It nicely separates files from code, which I hope makes the docs that much easier to scan/read.